py/picca/delta_extraction/utils_pk1d.py

```diff
@@ -162,6 +164,7 @@ def spectral_resolution_desi(reso_matrix, lambda_):
             4.0 / 2.0 / np.log(
                 reso[len(reso) // 2][:] / reso[len(reso) // 2 - 2][:]))) / 2.0
 
-    reso_in_km_per_s = (rms_in_pixel * SPEED_LIGHT * delta_lambda)
+    reso_in_AA = (rms_in_pixel * delta_lambda)
+    reso_in_km_per_s = reso_in_AA * SPEED_LIGHT / lambda_
 
-    return reso_in_km_per_s
+    return reso_in_km_per_s, reso_in_AA 
```

Comment:
See the related point in DesiData

Reply:
Here I would definitely recommend to just print out both for flexibility. It's a relatively crude estimate anyway.